tmail-backend/jmap/extensions/pom.xml

```diff
@@ -58,6 +58,10 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
We tend to sort dependencies alphabetically by groupId then by artifactId

Reply:
Let use ```mvn sortpom:sort``` for quickly sort this.